chore: default value for departmentName before sync to legacy (#322)
pagopa/io-services-cms: .changeset/wise-onions-prove.md

---
"io-services-cms-webapp": patch
---

> Placeholder value for field department name before syncing a service to legacy who lack's of

Reviewer: ```suggestion
Add a placeholder value before syncing a service to legacy one that lacks the field "department name"
```
